core/meta/inc/TClass.h

```diff
@@ -22,17 +22,23 @@
 
 #include "TDictionary.h"
 #include "TString.h"
+
+#ifdef R__LESS_INCLUDES
+class TObjArray;
+#else
 #include "TObjArray.h"
-#include "TObjString.h"
+
+// Not used in this header file but known to be relied upon by user code.
```

Comment:
And yet we don't `#include` them even without `R__LESS_INCLUDES`?

Reply:
It is suggestion of @pcanal from [this document](https://docs.google.com/document/d/1-IheLc7ekS9dboHdWXK1zmbO-rtVCvogP2pYFvFlX5k/edit)

Reply:
Then please change the comment. As is the message is that we are purposefully breaking user code. Maybe:
```suggestion
// Not used in this header file; user code should #include this directly.
```

Reply:
Same here - I rebase code instead of adding new commits